internal/controller/apiextensions/claim/reconciler.go

```diff
@@ -586,6 +586,7 @@ func Waiting() xpv1.Condition {
 		Type:               xpv1.TypeReady,
 		Status:             corev1.ConditionFalse,
 		LastTransitionTime: metav1.Now(),
-		Reason:             ReasonWaiting,
+		Reason:             xpv1.ConditionReason("Waiting"),
+		Message:            ReasonWaiting,
```

Comment:
```suggestion
		Message:            MessageWaiting,
```

I'd probably either rename this constant or just add the message string here inline (instead of using a constant).

Reply:
I've moved the string inline instead of using a constant.